src/Paysera/WalletApi/Client/WalletClient.php

```diff
@@ -1017,4 +1017,15 @@ public function deleteTransferConfiguration($clientId)
 
         return $this->mapper->decodeClient($responseData);
     }
+
+    /**
+     * @param $clientId
+     * @return Paysera_WalletApi_Entity_ClientPermissionsToWallet[]
+     */
+    public function getPermissionsToWallets($clientId)
+    {
+        return $this->mapper->decodeClientPermissionsToWallets(
+            $this->get(sprintf('client/%s/permissions-to-wallets', $clientId))
```

Comment:
is it ok, that there such endpoint doesn't exist?

Reply:
It exists in branch SUPPORT-84787_add_oauth_permission_manager in wallet-api project. We will deploy it before merging this PR.